nf_core/lint/schema_lint.py

```diff
@@ -88,4 +89,15 @@ def schema_lint(self):
         except AssertionError as e:
             warned.append(str(e))
 
+    # Check for mimetype in the 'input' parameter, warn if missing
+    if self.schema_obj.schema is not None:
+        try:
+            has_valid_mimetype = self.schema_obj.check_for_input_mimetype()
+            if has_valid_mimetype:
+                passed.append("Input mimetype lint passed")
```

Comment:
```suggestion
                passed.append(f"Input mimetype lint passed: {has_valid_mimetype}")
```
And maybe add it here